core/src/types/scheme.rs

```diff
@@ -249,6 +251,8 @@ impl Scheme {
             Scheme::Postgresql,
             #[cfg(feature = "services-gdrive")]
             Scheme::Gdrive,
+            #[cfg(feature = "services-ocios")]
+            Scheme::OciOs,
```

Comment:
Please use `Ocios` instead. We following the same pattern across the whole project:

- `Ocios` -> `ocios`
- `OciOs` -> `oci_os`